src/data/multiset/basic.lean

```diff
@@ -2042,6 +2042,10 @@ iff_not_comm.1 $ count_pos.symm.trans pos_iff_ne_zero
 theorem count_ne_zero {a : α} {s : multiset α} : count a s ≠ 0 ↔ a ∈ s :=
 by simp [ne.def, count_eq_zero]
 
+theorem one_le_count_iff_mem {x : α} {a : multiset α} : 1 ≤ a.count x ↔ x ∈ a :=
+⟨count_ne_zero.1 ∘ one_le_iff_ne_zero.1 ,
+  λ h, count_singleton_self x ▸ count_le_of_le x $ singleton_le.2 h⟩
```

Comment:
Turns out that this basically already exists as  `count_pos`: 
```suggestion
by rw [succ_le_iff, count_pos]
```
Can you move this lemma to right below `count_pos`?

Reply:
Done!